_cucumber/features/support/hooks.rb

```diff
@@ -59,6 +59,11 @@
 end
 
 def user_logout
-  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout?redirect_uri=#{@redirect_url}%3Fredirect_fragment%3D!")
+  if Capybara.app_host.include?('stage') || Capybara.app_host.include?('docker')
```

Comment:
I think what your are really doing here is testing if the environment is live? If that's the case, then test for the live URL as it is clearer and will allow us to use other forms of non-live